### ISSUE TITLE
It still works

### ISSUE THREAD

**Issue by alexk111** [alexk111/DeBitpay#23]
If you've recently paid a decoded invoice, please write a comment below so that others could see that it still works.

**Comment by MarlonGroenendijk** [comment]
Still works :)

**Comment by Laiteux** [comment]
Works.

**Comment by ordimans** [comment]
Can you add ETH ?

**Comment by jeanolivier225** [comment]
Still works ;)
Thanks guys

**Comment by jteib** [comment]
Still works!!

**Comment by adavis-threatkey** [comment]
still works, why is bitpay so annoying

**Comment by motsu35** [comment]
still working, thanks for making this. Why is bitpay so annoying

**Comment by AFTheMiddle** [comment]
still working

**Comment by openvpnhub** [comment]
Working well

**Comment by Kruwed** [comment]
It still works.  Fuck Bitpay.

**Comment by sillyh4ck3r** [comment]
Yep, still works.

**Comment by billyjeen22** [comment]
Does this still work? I can't find a bitcoin: URL on any of the bitpay payment options... The invoice url doesn't seem to work either.

**Comment by alexk111** [comment]
@billyjeen22 Please create a new issue with this url: https://github.com/alexk111/DeBitpay/issues/new and provide more details such as the link to the shop/site.

**Comment by Oleekae** [comment]
Hi, I have a question, I went to pay for a transaction to buy a service via Bitpay and once I select my wallet in the options it forces me to go through its KYC. The question is if it is impossible to get the Bitpay payment address without going through the KYC?

**Comment by alexk111** [comment]
@Oleekae If Bitpay forces you to go through KYC, it'll be not possible to get the payment address without KYC.

**Comment by Oleekae** [comment]
Thanks alexk111. 

It's really a rubbish service, im talking about small payments of 10$ or 15$. Really ridiculous.

**Comment by alexk111** [comment]
@Oleekae Agree. Try to share the bitpay experience with the merchant that uses it and ask if it's possible to pay without bitpay. If the merchant asks for better options - here is a list: https://github.com/alexk111/awesome-bitcoin-payment-processors/

**Comment by Oleekae** [comment]
I did so and he kindly replied that they did not at that moment consider any other option. I suggested implementing BTCPay as a payment platform.

**Comment by literally1984** [comment]
it worked, thank u so much for making this tool 😁

**Comment by turtleturtle1980-lab** [comment]
Don't work 18/02/2026  
"Error
×
Please log in to get payment details"

**Comment by tehradamant** [comment]
2/25/2026
Just works

**Comment by alexk111** [comment]
@turtleturtle1980-lab Could you resolve this? If not, please create a new issue with details.
